Backend/app.py

```diff
@@ -301,6 +301,27 @@ def updateClassDetails():
       status = 400
   return ResponseCreation.createResponse(res,status)
 
+@app.route('/forgotPassword/', methods=['POST', 'OPTIONS'])
+@handleOptionsRequest
+def forgotPassword():
+  status = 200
+  res = ResponseCreation.ControllerResponse()
+  dynamoDBInstance = getDatabaseClient()
+  httpOrigin = request.environ.get('HTTP_ORIGIN')
+  res = user_ctrl.sendForgotPasswordEmail(httpOrigin, request.json, mail, dynamoDBInstance)
```

Comment:
This function could fail after making the updates Micah pointed out. Remember to update this to handle if res.hasErrors()